food-truck-api/src/test/java/food/truck/api/endpoint/EndpointTest.java

```diff
@@ -31,6 +33,8 @@
 public class EndpointTest {
     @Autowired
     protected TestData data;
+    @MockBean
+    protected AmazonS3 s3Client;
```

Comment:
We may want to drop a comment explaining that this is to prevent autowiring AmazonS3 from failing in a testing environment.

Reply:
Added the comment